src/site/content/en/blog/optimizing-content-efficiency-eliminate-downloads/index.md

```diff
@@ -23,9 +25,38 @@ The fastest and best-optimized resource is a resource not sent. You should elimi
 - Site B has decided to install a third-party widget to display related content, improve social engagement, or provide some other service.
-  - **Question:** Have you tracked how many visitors use the widget or click-through on the content that the widget provides? Is the engagement that this widget generates enough to justify its overhead?
+  - **Question:** Have you tracked how many visitors use the widget or click-through on the content that the widget provides? Is the engagement that this widget generates enough to justify its overhead? Furthermore, is it feasible for you to use a loading strategy to ensure [the script isn't loaded until it's needed](/reduce-javascript-payloads-with-code-splitting/)?
 
 Determining whether to eliminate unnecessary downloads often requires a lot of careful thinking and measurement. For best results, periodically inventory and revisit these questions for every asset on your pages.
 
+## Effects on Core Web Vitals
+
+The Core Web Vitals initative was introduced by Google to provide a set of metrics that accurately reflect what users are experiencing as they use the web. While there are many optimization strategies for Core Web Vitals, questioning whether to load a particular resource on a page may light a path for you to improve these metrics on your website. Below are a few examples—grouped by each Core Web Vital—that are worth your consideration. Though this isn't an exhaustive list of examples (and there are many!), reading them over may give you food for thought.
+
+### Largest Contentful Paint (LCP)
+
+[Largest Contentful Paint (LCP)](/lcp/) measures when the largest text node, image, or video with a `poster` image loads. It's considered an important perceptual metric that gives the user the impression that a site is loading quickly.
+
+In general, downloading less resources means that the bandwidth the user does have will be allocated across less resources, and may translate to an improvement in LCP. A classic example is that of lazy loading, where images outside of the viewport during page load will not be downloaded until the browser has determined the user is more likely to see them. If you have a large thumbnail gallery of, say, 50 images, lazy loading all but the top ten of them means that the browser can make more efficient use of the bandwidth available to it, and the remaining images will load more quickly.
+
+However, it's not just about loading less _images_, necessarily. While the browser has an internal prioritization scheme that determines how much bandwidth each resource should receive, _all_ resources—particularly those downloaded at high priority—have the potential to deprive a potential LCP element's dependent resource, especially on slow network connections. That dependent resource may be an image file that represent's the pages LCP element—but it could also very well be a web font resource that the browser needs to render a text node that may be determined as the page's LCP element.
```

Comment:
That second sentence has a lot of sub-clauses. Can you split it up at all?

```suggestion
However, it's not just about loading less _images_, necessarily. The browser has an internal prioritization scheme that determines how much bandwidth each resource should receive. However, even with this _all_ resources—particularly those downloaded at high priority—have the potential to deprive a potential LCP element's dependent resource. This is especially true on slow network connections. That dependent resource may be an image file that represent's the pages LCP element but it could also very well be a web font resource that the browser needs to render a text node that may be determined as the page's LCP element.
```